infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -81,6 +81,11 @@
     private CorfuMsgHandler handler = new CorfuMsgHandler()
             .generateHandlers(MethodHandles.lookup(), this);
 
+    @Override
+    public boolean isServerReady() {
```

Comment:
All servers have different implementations.
Base, Layout and Management Servers are by default in the ready state so their implementations return true directly.
The Logunit and the Sequencer servers start in a non-ready state and have to wait for the isReady flag to be true.